package.json

```diff
@@ -36,18 +36,19 @@
   },
   "scripts": {
     "build-storybook": "storybook build",
-    "build": "lingui compile && vite build --mode lib && vite build --mode eik",
+    "build": "vite build --mode lib && vite build --mode eik",
     "commit": "cz",
     "dev": "vite dev dev --config ./vite.config.js",
     "lint:check": "eslint . --ext js,vue,cjs,mjs --ignore-path .gitignore",
     "lint": "eslint . --fix --ext js,vue,cjs,mjs --ignore-path .gitignore",
     "messages:compile": "lingui compile",
     "messages:extract": "lingui extract",
+    "postinstall": "lingui compile",
     "postsite": "cd dev/dist && cp index.html 200.html && cp index.html 404.html",
     "site": "vite build dev --config ./vite.config.js",
     "storybook": "storybook dev -p 6006",
-    "test": "lingui compile && vitest run --coverage",
-    "watch": "lingui compile && vitest watch"
+    "test": "vitest run --coverage",
+    "watch": "vitest watch"
```

Comment:
Is `postinstall` going to run before tests?

Reply:
Yes! All of the build-steps passed already in this draft-PR, and you can also see here that `postinstall` runs before the tests: 
<img width="1440" alt="Skärmavbild 2024-12-02 kl  13 35 56" src="https://github.com/user-attachments/assets/a279afd2-9990-4b9e-9dd0-c2ea5cb382f2">